### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by dibarbet**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,8 +4,14 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
+* Fix `Attribute` snippet (PR: [#6879](https://github.com/dotnet/vscode-csharp/pull/6879))
+* Add grammar support for new Razor directives: rendermode, preservewhitespace and typeparam (PR: [#6887](https://github.com/dotnet/vscode-csharp/pull/6887))
+* Add Razor inlay hint and inlay hint resolve handlers (PR: [#6857](https://github.com/dotnet/vscode-csharp/pull/6857))
+* Ensure failures to connect to the named pipe are propogated (PR: [#6881](https://github.com/dotnet/vscode-csharp/pull/6881))
 * Update Razor to 7.0.0-preview.24079.3 (PR: [#6841](https://github.com/dotnet/vscode-csharp/pull/6841))
   * Fix an issue with component hover not showing enough information (PR: [dotnet/razor#9862](https://github.com/dotnet/razor/pull/9862))
```
Comment (jasonmalinowski) [inline CHANGELOG.md:12]: Did this ultimately not make it into 2.18.15 release?

**Review by jasonmalinowski — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,8 +4,14 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
+* Fix `Attribute` snippet (PR: [#6879](https://github.com/dotnet/vscode-csharp/pull/6879))
+* Add grammar support for new Razor directives: rendermode, preservewhitespace and typeparam (PR: [#6887](https://github.com/dotnet/vscode-csharp/pull/6887))
+* Add Razor inlay hint and inlay hint resolve handlers (PR: [#6857](https://github.com/dotnet/vscode-csharp/pull/6857))
```
Comment (davidwengier) [inline CHANGELOG.md:9]: Thanks for adding this, but without a razor insertion they don't actually do anything. Was gonna mention it when we insert. Up to you if you want to leave it though, the statement is technically correct :)

Comment (dibarbet) [reply CHANGELOG.md:9]: Ah I didn't realize, will remove

File: CHANGELOG.md (hunk 1 above)
Comment (dibarbet) [reply CHANGELOG.md:12]: yeah - just got added to latest in main before I updated the last released version numbners

**Pull request merged**